dispatcher/service/pool.py

```diff
@@ -110,10 +108,9 @@ def __init__(self) -> None:
 
 
 class WorkerPool:
-    def __init__(self, max_workers: int, process_manager: ProcessManager, settings: LazySettings = global_settings):
+    def __init__(self, max_workers: int, process_manager: ProcessManager):
```

Comment:
This is going to conflict hard with the autoscaling PR.

But this is nice, because this bumps us down 1 kwarg, and that adds a ton of scalar kwargs. `WorkerPool.__init__` is one of the most modified method signature around, lol.